pages/api/randomProfile.js

```diff
@@ -0,0 +1,40 @@
+// pages/api/randomProfile.js
+require('dotenv').config();
+
+
+const { MongoClient } = require('mongodb');
+
+// console.log('MONGODB_URI:', process.env.MONGODB_URI);
+
+
+const uri = process.env.MONGODB_URI;
+const client = new MongoClient(uri, { useNewUrlParser: true, useUnifiedTopology: true });
```

Comment:
Not sure why you are creating another client here, we have one available to use already, please see other api files

Reply:
Modified and used the readily available